org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/jres/JREsPreferencePage.java

```diff
@@ -167,6 +174,9 @@ public void selectionChanged(SelectionChangedEvent event) {
 				}
 			}
 		});
+
+		SWTFactory.createVerticalSpacer(ancestor, 1);
```

Comment:
This preference is hardly detectable at the bottom of the table, it should be *above* the table with available JRE's

Reply:
Ack.